ci/test/cargo-test/image/run-tests

```diff
@@ -20,7 +20,30 @@ ci_init
 
 while read -r binary cwd; do
     cd "$cwd"
-    ci_try "/tests/$binary" --nocapture
+    ci_try "/tests/$binary" --nocapture -Z unstable-options --format json --report-time | tee results.json
```

Comment:
So, once the other PR lands, I think only this line should stay in this file, and probably wants to become:

```
(cd "$cwd" && ci_try "/tests/$binary" --nocapture -Z unstable-options --format json --report-time | tee -a results.json)
```